OMEdit/OMEditGUI/Modeling/LibraryTreeWidget.cpp

```diff
@@ -1010,6 +1010,9 @@ bool LibraryTreeProxyModel::filterAcceptsRow(int sourceRow, const QModelIndex &s
     if (mShowOnlyModelica && pLibraryTreeItem && pLibraryTreeItem->getLibraryType() != LibraryTreeItem::Modelica) {
       return false;
     }
+    if (pLibraryTreeItem->getNameStructure().compare("OMEdit.Search.Feature") == 0) {
+         return false;
+    }
```

Comment:
I think we need to write a proper explanation here. Otherwise we will forget why we have added this here.